app/model.py

```diff
@@ -74,8 +86,8 @@ class CellMeta(Model):
     ah = Column(BigInteger, nullable=True)
     form_factor = Column(TEXT, nullable=True)
     test = Column(TEXT, nullable=True)
-    mapping = Column(TEXT, nullable=True)
     tester = Column(TEXT, nullable=True)
+    # mapping = Column(TEXT, nullable=True)
```

Comment:
Can we delete this?

Reply:
mapping is an interesting one. We should decide if we're going to support this usage pattern